modules/standard/fun/fun_controller.py

```diff
@@ -0,0 +1,78 @@
+        self.db: DB = registry.get_instance("db")
+        self.command_alias_service = registry.get_instance("command_alias_service")
+
+    def start(self):
+        self.command_alias_service.add_alias("pinky", "brain")
+        self.command_alias_service.add_alias("norris", "chuck")
+        self.command_alias_service.add_alias("chucknorris", "chuck")
+        self.command_alias_service.add_alias("cyb0r", "cybor")
+        self.command_alias_service.add_alias("cyber", "cybor")
+        self.command_alias_service.add_alias("office", "dwight")
+        self.command_alias_service.add_alias("funcom", "fc")
+        self.command_alias_service.add_alias("simpsons", "homer")
+
+    @command(command="beer", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random beer message")
+    def beer_command(self, request, item_id):
+        return self.get_fun_message("beer", item_id)
+
+    @command(command="brain", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random pinky and the brain quote")
+    def brain_command(self, request, item_id):
+        return self.get_fun_message("brain", item_id)
+
+    @command(command="chuck", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random Chuck Norris joke")
+    def chuck_command(self, request, item_id):
+        return self.get_fun_message("chuck", item_id)
+
+    @command(command="cybor", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random cybor message Caution: HOT!")
+    def cybor_command(self, request, item_id):
+        return self.get_fun_message("cybor", item_id)
+
+    @command(command="dwight", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random Dwight quote")
+    def dwight_command(self, request, item_id):
+        return self.get_fun_message("dwight", item_id)
+
+    @command(command="fc", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random FC quote")
+    def fc_command(self, request, item_id):
+        return self.get_fun_message("fc", item_id)
+
+    @command(command="homer", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random homer quote")
+    def homer_command(self, request, item_id):
+        return self.get_fun_message("homer", item_id)
+
+    @command(command="pirates", params=[Int("item_id", is_optional=True)],
+             access_level="member", description="Shows a random Pirates of the Caribbean quote")
+    def pirates_command(self, request, item_id):
+        return self.get_fun_message("pirates", item_id)
+
+    def get_fun_message(self, quote_type, number):
+        if number is None:
+            number = -1
+        data = self.db.query("SELECT f.content FROM fun f WHERE type = ?", [str(quote_type)])
```

Comment:
If you add an id to the fun table, you should be able to avoid having to fetch _all_ the quotes, when you really only want one.